projects/addon-charts/components/pie-chart/pie-chart.component.ts

```diff
@@ -104,9 +99,7 @@ export class TuiPieChartComponent {
     }
 
     get mask(): string | null {
```

Comment:
Should we link this PR to this issue ?
https://github.com/Tinkoff/taiga-ui/issues/4769